tex/generic/pgf/libraries/luamath/pgflibraryluamath.code.tex

```diff
@@ -406,7 +406,7 @@
 \def\pgfluamathadd@#1#2{%
   \pgfluamathgetresult{pgfluamathfunctions.add(#1,#2)}}%
 \def\pgfluamathsubtract@#1#2{%
-  \pgfluamathgetresult{pgfluamathfunctions.subtract(#1,#2)}}%
+  \pgfluamathgetresult{pgfluamathfunctions.substract(#1,#2)}}%
```

Comment:
Why did you revert this?  Is `substract` instead of `subtract` actually correct?

Reply:
See https://github.com/pgf-tikz/pgf/pull/711#issuecomment-513389257